### PR TITLE
Fix android sdk sources path

### PR DESCRIPTION
**Pull request by sooth-sayer**

### CODE REVIEW

File: autoload/android.vim [hunk 1]
```diff
@@ -248,9 +248,10 @@ function! android#setClassPath()
     return
   endif
 
-  if exists(":JCstart")
```
Comment (DonnieWest) [inline autoload/android.vim]: The reason for this conditional is checking for JavaComplete2, which will manage classpaths for us. Do you have a good reason to remove it?

Comment (sooth-sayer) [inline autoload/android.vim]: @DonnieWest JavaComplete2 does not add target android platform sources path and jar path

Comment (DonnieWest) [inline autoload/android.vim]: @sooth-sayer It should, would you mind opening an issue in the JavaComplete2 repo? I can hopefully fix it from there once I have more information

Comment (sooth-sayer) [inline autoload/android.vim]: @DonnieWest You're right! I missed g:JavaComplete_SourcesPath option

**Comment by hsanson** [comment]
Good catch, thanks...  I need to start implementing unit tests for the plugin..

**Pull request merged**